task/control-plane-agent/src/mgs_psc.rs

```diff
@@ -26,7 +26,8 @@ use userlib::sys_get_timer;
 
 // How big does our shared update buffer need to be? Has to be able to handle SP
 // update blocks for now, no other updateable components.
-const UPDATE_BUFFER_SIZE: usize = SpUpdate::BLOCK_SIZE;
+const UPDATE_BUFFER_SIZE: usize =
```

Comment:
This is a drive-by fix to make `UPDATE_BUFFER_SIZE` more robust in case per-component block size ever diverges; `mgs_gimlet.rs` was already doing this.